### DISCUSSION TITLE
Trying to get listings and their sales of other sellers

### DISCUSSION THREAD

**Discussion by RodiCoci** [etsy/open-api#1198] [Q&A]
Hello, 

I would like to read data from other shops, like:
- their listings info
- their sales per listing

I now have a application for personal use. When I try to call the API i get error 401. This means I don't have the right access rights, correct? 

How do I get access to this info? Do I have to apply for a commercial application? But I don't want to commercialize this application, I just want to use this for myself.

Can anyone please help me out? 

Many thanks in advance,
Pieter

**Comment by Gareth064** [comment]
I believe listing information is public, so you should be able to read that.

The only way to get the number of sales for a listing is to have access to sales data, and for that you need the owner of the shop to authorize your access to their shop.

**Reply by RodiCoci** [reply]
Which API reference should I use then? I used this one and it didn't work: 'https://developer.etsy.com/documentation/reference/#operation/getListingsByShop'

I get the error 'Error: User does not own Shop'.

How do applications like Alura and everbee get access to these sales data? 

Thanks in advance

**Reply by Gareth064** [reply]
The endpoints beginning with "find" are normally public.

![Screenshot_20240126-175008.png](https://github.com/etsy/open-api/assets/22307223/0ddd4011-764a-494e-b7bb-5f387149b494)

I can't tell you how a lot of those other sites get half the data they do. Most of it isn't available through the API so I can only imagine they are doing some sort of scraping.

**Comment by showtroylove** [comment]
They obtain their data using the same API namely, Etsy. I offer the same information in my app. They're not the **actual** sales data they're estimates only. As a commercial app, you can have shop owners authorize your app, but this would likely be for the benefit of the shop owner and perhaps key staff.
